x/bank/keeper/keeper_test.go

```diff
@@ -35,7 +35,7 @@ var (
 	multiPermAcc  = authtypes.NewEmptyModuleAccount(multiPerm, authtypes.Burner, authtypes.Minter, authtypes.Staking)
 	randomPermAcc = authtypes.NewEmptyModuleAccount(randomPerm, "random")
 
-	initTokens = sdk.TokensFromConsensusPower(initialPower)
+	initTokens = sdk.TokensFromConsensusPower(initialPower, sdk.DefaultPowerReduction)
```

Comment:
`initTokens` variable is used in several places, to be secure, variable name should be `defaultInitTokens` as it's derivated from `sdk.DefaultPowerReduction`.

Reply:
I think that because these are in tests, I think it's okay as is.  If this was in state machine codepaths, I'd agree.  But this is just a variable name local to the test, and so like to the view of the test, it is _the_ initTokens, not just a default InitTokens.

Ditto to the other similar comments.

Reply:
`initTokens` is a global varible used in tests, it's not local function variable. It can be used in several tests where power reduction could be different.

Reply:
Hmm, ok

Reply:
For my perspective, global var `initTokens` can be initialized from DefaultPowerReduction and local `initTokens` can be initialized from on-chain power reduction. If local test has different on-chain power reduction, it can initialize local `initTokens`.